api-catalog-services/src/main/resources/application.yml

```diff
@@ -45,13 +45,13 @@ mfaas:
             homePage: /ui/v1/apicatalog
         info:
             serviceTitle: API Catalog
-            description: API Catalog microservice to display service details and API documentation for discovered API services
+            description: API Catalog service to display service details and API documentation for discovered API services.
             enableApiDoc: true
     service:
         hostname: ${environment.hostname}
         ipAddress: ${environment.ipAddress}
     catalog-ui-tile:
-        id: apicatalog
+        id: apimediationlayer
```

Comment:
why do we change `apicatalog` to `apimediationlayer` ?

Reply:
because the tile will no longer be for just the catalog but all internal services for the core mediation layer services - catalog, gateway, discovery (if we choose). This is an internal, for our use, not for modification by others.